PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -237,5 +260,18 @@ public static void PasteToPosition(Models.PowerPointSlide slide, bool clipboardI
                 }
             }
         }
+
+        private static ShapeRange RemovePlaceholders(PowerPointSlide slide, ShapeRange shapes)
```

Comment:
I agree, Graphics contains many shape related methods. They should be put into a separate util class